docs/pages/deployment/cli-reference.rst

```diff
@@ -1,29 +1,18 @@
 .. _nuts-cli-reference:
 
 CLI Command Reference
-*********************
+#####################
 
+There are 2 types of commands: server command and client commands. Server commands (e.g. ``nuts server``) can only be run on the system where the node is (or will be) running, because they require the node's config. Client commands are used to remotely administer a Nuts node and require the node's API address.
```

Comment:
Most of these flags don't do anything for `nuts crypt fs2vault` and the `nuts http gen-token` right? To me it is still not clear what flags are required to successfully use these commands.
This does improve readability though.

Reply:
The current commands of the engines,  that use the server config, don't have additional flags, other than those here. If we introduce a command that does, it will not show up in these docs. That's an issue, but to be solved when we need it.